### DISCUSSION TITLE
Compatible with spatie/laravel-permission

### DISCUSSION THREAD

**Discussion by mjhcremer** [Power-Components/livewire-powergrid#798] [Q&A] [answered]
Is it possible to check if a user has an certain permission so the button can be disabled?

**Comment by dansysanalyst** [comment, marked as answer]
You can use the [Action Rules](https://livewire-powergrid.com/table/action-rules.html#modifiers)

It would be something like:

```php
Rule::button('delete')
    ->when(fn() =>  auth()->user()->can('delete_stuff') === false)
    ->disable(),
```
